web/themes/custom/novel/templates/layout/eventinstance--full.html.twig

```diff
@@ -19,6 +19,10 @@
   {
     label: "Address"|trans,
     value: content.event_address.0 ? content.event_address : null
+  },
+  {
+    label: "branch"|trans,
+    value: content.branch.0 ? content.branch : null
```

Comment:
Please discuss this addition with the business.

I think I understand your reasoning but there is some duplication between the address field (which we have not figured out how to handle with multiple branches), the event partners and the branches.

As I see it we are not showing the branch because it will duplicate the address.

Reply:
I have removed it for now in:
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/878/commits/1a2067e9ea3c576c06cd9d2a5482ab34f4c12c57